wdl-lint/tests/lints/curly-command/source.errors

```diff
@@ -1,3 +1,11 @@
+note[SectionOrdering]: sections are not in order for task bad
```

Comment:
I think we should add an `#@ except` for this lint to the existing files and create a dedicated test for the lint.